### PR TITLE
fixing the padding at cards' name and add github and linkedin button

### PR DESCRIPTION
**Pull request by blinfoldking**

### CODE REVIEW

File: member.html [hunk 1]
```diff
@@ -70,6 +70,10 @@ <h1 class="title">
                       <h4 class="title is-4">Wisnu Adi Nurcahyo</h4>
                       <h6 class="subtitle is-6">Co-Founder, IF'16</h6>
                     </div>
+                    <footer class="card-footer">
```
Comment (wisn) [inline member.html]: Use `div` instead of `footer` since we just need one `footer` here.

File: assets/css/default.css [hunk 2]
```diff
@@ -41,3 +41,7 @@ footer .icons a {
   display: inline-block;
   margin: 0 .25em;
 }
+
+.title {
+  padding: 20px;
```
Comment (wisn) [inline assets/css/default.css:46]: Why we need this? Could you give the screenshot?

I think we need to setup a CI here for preview.

**Review by wisn — CHANGES_REQUESTED** [review]

File: assets/css/default.css (hunk 2 above)
Comment (blinfoldking) [reply assets/css/default.css:46]: screenshot from 2018-05-11 23-29-01

Comment (wisn) [reply assets/css/default.css:46]: Actually, the main problem is the `clip-path` property. However, we could accept it now and we need to fix it ASAP.

**Review by wisn — APPROVED** [review]

**Pull request merged**